packages/wdio-reporter/src/types.ts

```diff
@@ -0,0 +1,611 @@
+import * as got from 'got'
```

Comment:
I suggest to update the package `tsconfig.json` to include types from the `webdriver` package once I merged my PR.